app/data/models/invoice-line.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (sequelize, DataTypes) => {
+  const invoiceLine = sequelize.define('invoiceLine', {
+    invoiceLineId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+    paymentRequestId: DataTypes.INTEGER,
+    schemeCode: DataTypes.STRING,
```

Comment:
Can acc'ly give explicit string length to match what the DB has, i.e. `varchar(6)` should this be `DataTypes.STRING(6)` see here: https://sequelize.org/docs/v7/other-topics/other-data-types/

Reply:
Yeah that's fine.  Just a sequelize limitation.  At it cares about is the JS data type to cast is as.  Any issues with length, precision etc are handled as DB errors rather than model errors.